sql/catalyst/pom.xml

```diff
@@ -60,6 +60,11 @@
       <artifactId>scalacheck_${scala.binary.version}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I'd like to have all the information of janino in one place, it's easy to manage.